### ISSUE TITLE
Getting 0 score while querying similar products

### ISSUE THREAD

**Issue by sameersharma00747** [RedisAI/vecsim-demo#12]
What's the reason for getting zero as score while querying similar vector in Redis. All results returned are having 0 as score.

**Comment by alonre24** [comment]
Can you describe the use case in more details? How did you create your index? what was your query?

**Comment by sameersharma00747** [comment]
Actually I created 0.5 million random vectors of dimension 100. I used numpy.random.random() to generate 100 dimension vectors. As a primary key I had passed randomly generated product IDs. At the query stage as well, I was passing a 100 dimensional vector as a query vector.

**Comment by DvirDukhan** [comment]
@sameersharma00747 are you using an apple-silicon machine? please notice this issue https://github.com/RediSearch/RediSearch/issues/2725

**Comment by sameersharma00747** [comment]
@DvirDukhan Yes, I am running it on Apple M1 pro. Also, I am using L2 as the distance metric.

**Comment by DvirDukhan** [comment]
Can you please try it with Redis stack installation? The docker is x86 build so it is not compatible with m1.

**Comment by sameersharma00747** [comment]
Sure, will try that. Hope it works.
